### ISSUE TITLE
Grids on mobile

### ISSUE THREAD

**Issue by marvoh** [imperavi/kube#65]
Hi,

I've rummaged through the documentation but couldn't find a way to target mobile screens with grids.

As it is, the columns automatically grow to 100% while I was looking at the option of say splitting a row into two on phones.

Let me know if there's a way to do this

**Comment by Limboistik** [comment]
Looking to do this as well!

For example like how Bootstrap or Foundation does it, say if I want a grid to be 6 cols on Tablets but 8 col on Desktop, and mobile defaults to full width, the markup would be:

`<div class="col-md-6 col-lg-8">`

**Comment by ScotsScripts** [comment]
I agree. Right now there's only one break point - big and small. If column breaking is going to be automatic, it would be cool to have a medium break point that would divide column numbers in half before dropping to one. 

Something like: 
3-3-3-3 

would break into:
6-6
6-6

**Comment by invasoria** [comment]
There was a way to do this on previous versions (two-columns grids for mobile view) but it seems to be deprecated. +1 for all of these comments. It would be great to have this functionality out of the box.

**Comment by Zmove** [comment]
+1 for this. I was very excited with kube, especially the class naming convention that is so clean.

It would be perfect but I will not use it cause of 2 things that is important to me :

- The ability to define custom grid depending on breakpoint. Not all design need to be in 1 column under 768px...
- The smallest breakpoint of 768px it much to large to me. No phone in portrait format (even in landscape for some of them) can reach this.

With that 2 features, I would definitely use kube cause the rest is perfect.

**Comment by Madebym** [comment]
Just to chime in. Is there any reason why this is still not implemented?

As said in the post above, everything else is perfect, why not add this(IMO a must have feature) to the grids?

There are many scenarios where having a one column grid on mobile is not good enough. I would really like to hear the reasons why this is left out.

**Comment by bgarrant** [comment]
I just opened another issue post with same question. I see we can use a mixin to create a breakpoint, but I need to be able to use two columns (col-xs-6) while on devices say 480 to 768px wide. Has anyone found a way? This is like the only issue I have had with Kube. The rest of the framework is fantastic.  Can anyone help override the stacking of columns on the 480 to 768px screens?

**Comment by bgarrant** [comment]
How about a class called "mobile" that just overrides the stacking for devices 480 to 768px? Any way to do this?

**Comment by bgarrant** [comment]
Would something like this work? Can't test as on my phone at moment but may work. 


#container {
    @include grid-row;
}
#mobile50 {
    @include flex-item-width(50%);
}


<div id="container">
    <div id="mobile50">50% column</div>
    <div id="mobile50">50% column</div>
</div>

**Comment by iiAtlas** [comment]
+1! I am having to drop Kube from my project due to the inability to target specific grid sizes (like Flexbox Grid etc), shame.

**Comment by kisabelle** [comment]
+1 This feature is a must for me.. I will be adding it in manually. It really feels like something that should be in there! The way Bootstrap does it is excellent in my experience.
